src/dex/maker-psm/maker-psm.ts

```diff
@@ -13,21 +13,22 @@ import {
 } from '../../types';
 import { SwapSide, Network } from '../../constants';
 import { StatefulEventSubscriber } from '../../stateful-event-subscriber';
-import { wrapETH, getDexKeysWithNetwork } from '../../utils';
+import { wrapETH, getDexKeysWithNetwork, getBigIntPow } from '../../utils';
 import { IDex } from '../../dex/idex';
 import { IDexHelper } from '../../dex-helper/idex-helper';
 import { MakerPsmData, PoolState, PoolConfig } from './types';
 import { SimpleExchange } from '../simple-exchange';
 import { MakerPsmConfig, Adapters } from './config';
 import PsmABI from '../../abi/maker-psm/psm.json';
 import VatABI from '../../abi/maker-psm/vat.json';
+import { BI_0, BI_1, BI_POWS } from '../../bigint-constants';
 
 const vatInterface = new Interface(VatABI);
 const psmInterface = new Interface(PsmABI);
-const WAD = BigInt(10 ** 18);
-const BN0 = BigInt(0);
-const BN1 = BigInt(1);
-const BN1E18 = BigInt(1e18);
+const WAD = BI_POWS[18];
+const BN0 = BI_0;
```

Comment:
Do we need additional aliases? can't we just use BI_0 wherever BN0 was used? same for next lines